pom.xml

```diff
@@ -14,14 +14,14 @@
     </properties>
 
     <repositories>
+		<repository>
```

Comment:
Looks like your formatting is wrong, make sure to format your code before pushing. See here for the correct IntelliJ formatter profile: https://github.com/kernitus/BukkitOldCombatMechanics/blob/master/.github/CONTRIBUTING.md

Reply:
Thank you. It slipped my eye.